src/attestation/Attester.chain.ts

```diff
@@ -44,18 +44,21 @@ export default class AttesterChain extends Attester implements IAttesterChain {
    *
    * @param validityDuration The duration in days for which the public key will be valid.
    * @param maxAttributes The maximum number of attributes that can be signed with the generated private key.
+   * @param keyLength The key length of the new key pair. Note that this key will only support credentials and claimer with the same key length.
    * @param keypairType The signature scheme used in the keyring pair, either 'sr25519' or 'ed25519'.
    * @returns A [[AttesterChain]] instance including a chain address and a public and private key pair.
    */
   public static async create(
     validityDuration?: number,
     maxAttributes = 70,
+    keyLength: 1024 | 2048 | 4096 = 1024,
     keypairType: KeypairType = 'sr25519'
```

Comment:
we might want to add an `option` parameter here.

Reply:
I you mean a type for the key length like the following, I agree. Or what do you mean by that?
```typescript
type KeyLength = 1024 | 2048 | 4096
```

Reply:
If you want to change `keypairType` you have to also specify the `keyLength`. That's not so nice...

Reply:
True, definitely would make it optional and move to the last parameter
```typescript
  public static async create(
    validityDuration?: number,
    maxAttributes = 70,
    keypairType: KeypairType = 'sr25519',
    keyLength: 1024 | 2048 | 4096 = 1024
  ): Promise<AttesterChain> {
```

Reply:
Also, I would propose to add the keylengths as a type, see below.

Reply:
But if you know want to specify the key length and not the type you have the same problem again?

I would add an `option` parameter that can contain both `keypairType` and `keyLength`. Both of them are optional and this way both of them can be passed optional.

Reply:
Should probably add that to `Attester.genKeyPair()` as well.